salt/modules/omapi.py

```diff
@@ -47,6 +47,10 @@ def _conn():
             __opts__.get('omapi.key', None))
     username = __pillar__.get('omapi.user',
             __opts__.get('omapi.user', None))
+    if key:
+        key = key.encode('UTF-8')
+    if username:
+        username = username.encode('UTF-8')
```

Comment:
Instead of using `encode` here, you should be using `salt.utils.stringutils.to_bytes()`. This will avoid an `AttributeError` if this value happens to already be a bytestring (though this is very unlikely).